src/service/traces/mod.rs

```diff
@@ -161,7 +169,7 @@ pub async fn handle_trace_request(
     // check memtable
     if let Err(e) = ingester::check_memtable_size() {
         log::error!(
-            "[TRACE] ingestion error while checking memtable size: {}",
+            "[TRACES:OTLP] ingestion error while checking memtable size: {}",
```

Comment:
Hey, do you think it would be helpful for debugging (later) to add the `req_type` in `[TRACES:OTLP]` log like `[TRACES:OTLP:HTTP_JSON]` or similar? or it wouldn't be much useful?

Reply:
both fine